docs/cse/automation-service/automation-service-use-cases.md

```diff
@@ -0,0 +1,173 @@
+---
+id: automation-service-use-cases
+title: Use Cases
+sidebar_label: Use Cases
+description: Learn use cases to create automations for different situations.   
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+{@import ../../reuse/automation-service-la-note.md}
+
+Following are use cases that describe how to create automations for different situations.
+
+## Simple use case: Enable an enrichment
+
+The following example adds an enrichment using the “IP Reputation V3” action from VirusTotal:
+
+1. Edit the VirusTotal OIF resource:
+   1. Click the **Configuration** button (gear icon) at the top of the UI.
+   1. Under **Integrations**, select **Automation**.
+   1. From the Automation screen, click **Manage Playbooks**.
+   1. Click **Integrations** in the navigation menu.
+   1. Select **VirusTotal OIF**.
+   1. Hover your mouse over the resource name and click the Edit button that appears.
+   1. Enter the **API URL**: https://www.virustotal.com
```

Comment:
I would deactivate the hyperlink since it's something customers will need to enter in the UI.

```suggestion
   1. Enter the **API URL**: `https://www.virustotal.com`.
```